.github/workflows/helm-multi-release.yaml

```diff
@@ -0,0 +1,197 @@
+name: Helm Multi Release
+# Reusable workflow for packaging and publishing Helm charts having multiple subcharts on GitHub pages
+
+on:
+  workflow_call:
+    inputs:
+      charts-path:
+        description: "The directory containing the Helm chart(s)."
+        required: true
+        type: string
+      subdirs:
+        description: "List of subdir to consider" # "['subdir1', 'subdir2', 'subdir3']"
+        required: true
+        type: string
+      artifact-dir:
+        description: "Directory next to `charts-path` for preparation of the GitHub pages artifact."
+        required: false
+        default: "artifacts"
+        type: string
+      gh-pages-branch:
+        description: "Name of branch containing the artifacts"
+        required: false
+        default: "gh-pages"
+        type: string
+      helm-version:
+        description: "The Helm version."
+        required: false
+        default: "v3.10.1"
+        type: string
+      lint-config-path:
+        description: "The path to the lint configuration file (See https://github.com/helm/chart-testing/blob/main/pkg/config/test_config.yaml)."
+        required: false
+        default: ".github/lint-config.yaml"
+        type: string
+
+    secrets:
+      github-username:
+        description: "The GitHub username for committing the changes."
+        required: true
+      github-email:
+        description: "The GitHub email for committing the changes."
+        required: true
+      github-token:
+        description: "The GitHub token for committing the changes."
+        required: true
+
+jobs:
+  helm-lint:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Lint Helm chart
+        uses: bakdata/ci-templates/actions/helm-lint@v1.10.0
+        with:
+          lint-config-path: "${{ inputs.lint-config-path }}"
+
+  update-version:
+    runs-on: ubuntu-22.04
+    outputs:
+      singlechart: ${{ steps.step2.outputs.singlechart }}
+    needs: [helm-lint]
+    env:
+      release-tag: ${{ github.ref_name }}
+      CHARTS: ${{ inputs.subdirs }}
+
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ env.release-tag }}
```

Comment:
Just remove `ref` completely so that we use the default (which is always the branch that triggered it)

Reply:
i tried to remove but it checkout the main branch, since this is the default branch. but i want to checkout tag. right?